src/sql/magic_cmd.py

```diff
@@ -33,6 +34,18 @@ def error(self, message):
 class SqlCmdMagic(Magics, Configurable):
     """%sqlcmd magic"""
 
+    displaycon = True
+    autolimit = None
+    style = "DEFAULT"
+    short_errors = True
+    displaylimit = None
+    autopandas = False
+    column_local_vars = False
+    feedback = False
+    autocommit = False
+    autopolars = False
```

Comment:
are these needed? these look like the config from the original sqlmagic and I don't think we're using them here

Reply:
I tried to pass `self.config` but it looks like there is no default configuration there and it fails to run the query.
e.g: `self.config.autocommit` returns `LazyConfigValue` (which implies that there is no such config attribute?)

I noticed that if we run `%sqlcmd columns` on a table loaded with DuckDB we get `NoSuchTableError`.

e.g:
```
%sqlcmd columns --table "yellow_tripdata_2021-01.parquet"
```

Is this may be related?

*Note
This is why I'm using SQL query to get column names instead `inspector.get_columns(name, schema)`.

Reply:
Ok, I think this will be fixed if we migrate to a `run_raw` function as described in my earlier comment

> I noticed that if we run %sqlcmd columns on a table loaded with DuckDB we get NoSuchTableError.

I think this is expected since the parquet is not a table; but can you open an issue? I think we should document it since it might confuse users